serenata_toolbox/chamber_of_deputies/reimbursements_cleaner.py

```diff
@@ -0,0 +1,173 @@
+    ('13', 'Congressperson meal'),
+    ('14', 'Lodging, except for congressperson from Distrito Federal'),
+    ('15', 'Automotive vehicle renting or watercraft charter'),
+    ('119', 'Aircraft renting or charter of aircraft'),
+    ('120', 'Automotive vehicle renting or charter'),
+    ('121', 'Watercraft renting or charter'),
+    ('122', 'Taxi, toll and parking'),
+    ('123', 'Terrestrial, maritime and fluvial tickets'),
+    ('137', 'Participation in course, talk or similar event'),
+    ('999', 'Flight ticket issue')
+)
+DTYPE = {
+    'txNomeParlamentar': np.str,
+    'idecadastro': np.str,
+    'nuCarteiraParlamentar': np.str,
+    'nuLegislatura': np.str,
+    'sgUF': np.str,
+    'sgPartido': np.str,
+    'codLegislatura': np.str,
+    'numSubCota': np.str,
+    'txtDescricao': np.str,
+    'numEspecificacaoSubCota': np.str,
+    'txtDescricaoEspecificacao': np.str,
+    'txtFornecedor': np.str,
+    'txtCNPJCPF': np.str,
+    'txtNumero': np.str,
+    'indTipoDocumento': np.str,
+    'datEmissao': np.str,
+    'vlrDocumento': np.float,
+    'vlrGlosa': np.str,
+    'vlrLiquido': np.float,
+    'numMes': np.str,
+    'numAno': np.str,
+    'numParcela': np.str,
+    'txtPassageiro': np.str,
+    'txtTrecho': np.str,
+    'numLote': np.str,
+    'numRessarcimento': np.str,
+    'nuDeputadoId': np.str,
+    'ideDocumento': np.str,
+}
+
+
+def parse_float(value):
+    return float(value.replace(',', '.'))
+
+
+CONVERTERS = {
+    'vlrRestituicao': parse_float,
+}
+KEY = 'document_id'
+AGGREGATED_COLS = {
+    'reimbursement_number': 'numbers',
+    'net_value': 'total_net_value',
+    'reimbursement_value': 'total_value',
+}
+
+
+class ReimbursementsCleaner:
+    """
+    Perform data cleaning tasks over a reimbursements CSV file.
+    """
+
+    def __init__(self, year, path):
+        self.year = year
+        self.path = path
+        self.data = None
+
+    def __call__(self):
+        self.load_source_file()
+        self.translate()
+        self.aggregate_multiple_payments()
+        self.save()
+
+    def load_source_file(self):
+        file_path = os.path.join(self.path, 'Ano-{}.csv'.format(self.year))
+        self.data = pd.read_csv(file_path,
+                                delimiter=';',
+                                quoting=csv.QUOTE_NONE,
+                                decimal=',',
+                                dtype=DTYPE,
+                                low_memory=False)
+
+    def translate(self):
+        self.data.rename(columns=COLUMNS, inplace=True)
+        for code, name in SUBQUOTAS:
+            rows = self.data['subquota_number'] == code
+            self.data.loc[rows, 'subquota_description'] = name
+
+    def aggregate_multiple_payments(self):
+        self.data = pd.concat([
+            self._house_payments(),
+            self._non_house_payments(),
+        ])
+
+    def save(self):
+        file_path = os.path.join(
+            self.path, 'reimbursements-{}.csv'.format(self.year))
```

Comment:
Two repeated comments: 

**1**

As we're using Python 3.6 we could make it a bit more readable using the new f-strings:

```python
file_path = os.path.join(self.path, f'reimbursements-{self.year}.csv')
```
**2**

Maybe here is a nice place to use `print(file_path)` to inform users where the dataset was actually saved (having in mind that it could be a random tmp dir).